### ISSUE TITLE
Double-line adds four spaces then using arrows to browse history

### ISSUE THREAD

**Issue by szczys** [diogocavilha/fancy-git#136]
Since the upgrade to 7.5.2 there is an anomaly when using double-line that adds four spaces after the prompt when using the up, then down arrows to browse command history: https://asciinema.org/a/Xm8rcFAb8edT2aNjWVG2NJSNN

## Reproduce

1. Turn on double-line: `fancygit --enable-double-line`
2. Type some commands
3. Use the up arrow to browse through some commands
4. Use the down arrow to browse the other way through commands
5. Four characters on the left of the command will be "stuck"

## System info

```
➜ uname -a
Linux krusty 5.15.0-67-generic #74-Ubuntu SMP Wed Feb 22 14:14:39 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
➜ echo $0
-bash
```

PS- thank you for this project, It's a wonderful addition to my terminal :heart: 

![image](https://user-images.githubusercontent.com/367685/232079593-feeea82d-fd72-4ba9-b31b-c4725bfe5833.png)

**Comment by szczys** [comment]
I have also discovered that your command must be 8 characters or greater to replicate this issue.

```
$ 1234567
# press up, then down; behaves as expected

$ 1234568
# press up, then down; 1234 will be "stuck"
```

**Comment by diogocavilha** [comment]
Hi, @szczys.

Thank you for reporting that. It's way detailed, thank you so much for that!

I've made some tests here, according to the steps you mentioned and I've discovered that it has something to do with the bold text prompt which has been released recently. 

Please, run `fancygit --disable-bold-prompt`. It must avoid the problem to keep happening. At least I did that here and it sopped happening. Please, let me know if it works for you as well.

I've created a branch already to implement a fix for that and probably it'll be solved on v7.5.3, which I'm working to release as soon as possible.

I'm so sorry for that. I know that these kind of bugs are annoying.

> PS- thank you for this project, It's a wonderful addition to my terminal :heart:

Oh thank you so much for that, for using Fancygit and for helping me to make it better :wink:

**Comment by diogocavilha** [comment]
Hey @szczys, I've just released the v7.5.3.

Please, let me know if you're still going through the same bug after upgrading it.

**Comment by szczys** [comment]
Apologies for the late reply. I tested both normal and bold prompts in two-line mode and #137 fixed the issue. Thank you!
